molecule/drupal8-db-dump/converge.yml

```diff
@@ -0,0 +1,141 @@
+  tasks:
+    - name: Include geerlingguy.php
+      include_role:
+        name: geerlingguy.php
+      vars:
+        php_enablerepo: "remi-php74"
+        php_default_version_debian: "7.4"
+        php_install_recommends: false
+        php_enable_php_fpm: true
+        php_webserver_daemon: "nginx"
+        php_date_timezone: "Europe/Paris"
+        php_expose_php: false
+        php_packages_extra: "{{ (ansible_os_family == 'Debian') | ternary(['php7.4-pgsql'], ['php-pgsql']) }}"
+    - name: Include anxs.postgresql
+      include_role:
+        name: anxs.postgresql
+      vars:
+        postgresql_users:
+          - name: "drupal"
+            pass: "drupal"
+        postgresql_databases:
+          - name: "drupal"
+            owner: "drupal"
+    - name: Include geerlingguy.git
+      include_role:
+        name: geerlingguy.git
+    - name: Include geerlingguy.drush
+      include_role:
+        name: geerlingguy.drush
+    - name: Include geerlingguy.nginx
+      include_role:
+        name: geerlingguy.nginx
+      vars:
+        nginx_ppa_use: true
+        nginx_remove_default_vhost: true
+        nginx_worker_processes: "{{ ansible_processor_vcpus|default(ansible_processor_count) }}"
+        nginx_vhosts:
+          - listen: "80"
+            server_name: "drupalll.test"
+            root: /drupal/project_root/www/web
+            extra_parameters: |
+              client_max_body_size 4096M;
+
+              location / {
+                  try_files $uri /index.php?$query_string;
+              }
+
+              location @rewrite {
+                rewrite ^ /index.php;
+              }
+
+              location ~ '\.php$|^/update.php' {
+                fastcgi_split_path_info ^(.+?\.php)(|/.*)$;
+                try_files $fastcgi_script_name =404;
+                include fastcgi_params;
+                fastcgi_param HTTP_PROXY "";
+                fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+                fastcgi_param PATH_INFO $fastcgi_path_info;
+                fastcgi_param QUERY_STRING $query_string;
+                fastcgi_intercept_errors on;
+                fastcgi_pass 127.0.0.1:9000;
+              }
+
+              location ~* \.(js|css|png|jpg|jpeg|gif|ico|svg)$ {
+                try_files $uri @rewrite;
+                expires max;
+                log_not_found off;
+              }
+
+              location ~ ^/sites/.*/files/styles/ {
+                try_files $uri @rewrite;
+              }
+
+              location ~ ^(/[a-z\-]+)?/system/files/ {
+                try_files $uri /index.php?$query_string;
+              }
+
+              if ($request_uri ~* "^(.*/)index\.php/(.*)") {
+                return 307 $1$2;
+              }
+
+              error_log /var/log/nginx/project_error.log;
+              access_log /var/log/nginx/project_access.log;
+        tags:
+          - steamengine_nginx
+    - name: Include steamulo.steamengine
+      include_role:
```

Comment:
Using include_role in a tasks whithout `tags: always` prevent usage of tag when developing / debugging with mocule. Particularly handy command like `molecule converge -s drupal8-db-dump -- -t steamengine_deploy_drupal`. 

An alternative to the tag always is the standard role usage with `roles:` in the playbook.